package.json

```diff
@@ -11,73 +11,59 @@
     "start": "npm run clean && npm run build && node dist/src/server.js",
     "build": "npm run clean && tsc -b",
     "clean": "tsc -b --clean",
-    "test": "TS_NODE_FILES=true tap --test-ignore='^dist'",
+    "test": "TS_NODE_FILES=true tap --ts --no-check-coverage --test-ignore='^(dist|scripts)'",
     "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix && tsc --noEmit",
     "prisma:generate": "prisma generate",
-    "prisma:migrate-dev": "prisma migrate dev"
+    "prisma:migrate-dev": "prisma migrate dev",
+    "updateDataDir": "scripts/update-data-dir.js"
```

Comment:
can potentially use this as a postinstall step if desired, but figured it wasn't necessary for now